types/connect-pg-simple/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for connect-pg-simple 4.2
+// Project: https://github.com/voxpelli/node-connect-pg-simple#readme
+// Definitions by: Pasi Eronen <https://github.com/pasieronen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import { RequestHandler } from "express";
+import { Store, SessionOptions } from "express-session";
+import { Pool, PoolConfig } from "pg";
+
+declare function connectPgSimple(session: (options?: SessionOptions) => RequestHandler): connectPgSimple.PGStore;
+
+declare namespace connectPgSimple {
+  interface PGStore {
+      new (options: PGStoreOptions): Store;
```

Comment:
`options` should be optional. https://github.com/voxpelli/node-connect-pg-simple/blob/master/index.js#L15